kv/split_test.go

```diff
@@ -108,7 +108,7 @@ func TestRangeSplitMeta(t *testing.T) {
 	}
 
 	util.SucceedsSoon(t, func() error {
-		if _, _, err := engine.MVCCScan(context.Background(), s.Eng, keys.LocalMax, roachpb.KeyMax, 0, hlc.MaxTimestamp, true, nil); err != nil {
+		if _, _, err := engine.MVCCScan(context.Background(), s.Eng, keys.LocalMax, roachpb.KeyMax, engine.MaxSpanKeys, hlc.MaxTimestamp, true, nil); err != nil {
```

Comment:
Do all of these packages already import `engine`? I don't think everyone should have to just to get a number.